src/formio/components/Password.js

```diff
@@ -14,6 +14,11 @@ class Password extends Formio.Components.components.password {
     );
     return info;
   }
+
+  getView(value, options) {
+    // In Formio they use '--- PROTECTED ---' which then somewhere gets stripped out
+    return '*'.repeat(value.length);
```

Comment:
I wouldn't use the length of the actual value.

```suggestion
    return '*'.repeat(12);
```